src/pages/Home/components/TeamBlock/TeamSlider/TeamSlider.tsx

```diff
@@ -0,0 +1,91 @@
+import { TeamPerson } from '@pages/Team/components/TeamPerson/TeamPerson';
+import React from 'react';
+import {
+  TABLET_POINT,
+  MOBILE_POINT,
+  MOBILE_SMALL_POINT,
+  TEAM_PERSONS
+} from '../../../../../constants';
+import { Swiper, SwiperRef, SwiperSlide } from 'swiper/react';
+import { Icon } from '@components/Icon/Icon';
+
+interface TeamSliderProps {}
+
+const TeamSlider: React.FC<TeamSliderProps> = () => {
+  const swiperRef = React.useRef<SwiperRef | any>(null); // delete the “any” type, how to solve the problem?
```

Comment:
Yes, you should remove any from the definition because it breaks all the typescript hints
"SwiperRef" is the correct type and it has swiper property you should use swiperRef.current?.swiper.slideNext() to switch slides